Microsoft.Toolkit.Uwp.UI.Controls/CameraPreview/CameraPreview.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    
+    <Style TargetType="local:CameraPreview" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:CameraPreview">
+                    <Grid Background="{TemplateBinding Background}">
+                        <MediaPlayerElement x:Name="MediaPlayerElementControl" HorizontalAlignment="Left">
+                        </MediaPlayerElement>
+                        <Button x:Name="FrameSourceGroupButton" Background="Transparent" 
```

Comment:
It might be better to have the FontIcon be light themed and the button to have a bit of a transparent dark background to make sure the button is visible in all situation. Can you change the Button Background to `{ThemeResource SystemBaseLowColor}` and the FontIcon Foreground to `{ThemeResource SystemAltHighColor}`